packages/node/src/utils/project.ts

```
@@ -42,14 +43,16 @@ export async function getChainTypes(
 ): Promise<ChainTypes> {
   // If the project is load from local, we will direct load them
   if (reader instanceof LocalReader) {
-    return loadChainTypes(file, root);
+    return loadChainTypes(file, root) as ChainTypes;
   } else {
     // If it is stored in ipfs or other resources, we will use the corresponding reader to read the file
     // Because ipfs not provide extension of the file, it is difficult to determine its format
     // We will use yaml.load to try to load the script and parse them to supported chain types
     // if it failed, we will give it another another attempt, and assume the script written in js
     // we will download it to a temp folder, and load them within sandbox
     const res = await reader.getFile(file);
+    assert(res, `File ${file} not found`);
```

I think `getFile` should be updated to throw rather than return undefined